docs/ref/models/fields.txt

```diff
@@ -796,7 +796,7 @@ Opens or reopens the file associated with this instance in the specified
 ``mode``. Unlike the standard Python ``open()`` method, it doesn't return a
 file descriptor.
 
-Since the underlying file is opened implicitly when accessing it, it may be
+Since the underlying file is opened implicitly when accessing it, it is
```

Comment:
or should this read:
```
Since the underlying file is only opened implicitly *on first access*, it may be
necessary call this method before accessing any other method.

Calling this method is also required to reset the pointer of the underlying
file or to change the ``mode``.
```